### PR TITLE
[JIT] Add support for closed over inf

### LINKED ISSUE #124: Handle -inf for Torchscript of FX graphs
Repro

~~~
import torch
import torch.fx

x = torch.randn(4, 5)
mask = torch.randn(4, 5) > 0.5

def f(x, mask):
    # return x.masked_fill_(mask, 1.0) # PASSES
    return x.masked_fill_(mask, float("-inf"))

print(f(x, mask))

# Only fails when symbolic_trace
fx_mod = torch.fx.symbolic_trace(f)
scripted_f = torch.jit.script(fx_mod)
print(scripted_f(x, mask))
~~~

@eellison

### PR DESCRIPTION
**Pull request by eellison**
Fixes https://github.com/facebookresearch/torchdynamo/issues/124

### CODE REVIEW

**Review by anijain2305 — APPROVED** [review]
Thanks for the quick change. Just confirming that it should work for `-inf` as we well, right?

**Review by davidberard98 — APPROVED** [review]

**Comment by eellison** [comment]
@anijain2305 yes, `-inf` is just -(positive infinity)
